sdk/lib/event_group/event_group.cc

```diff
@@ -176,3 +176,20 @@ int eventgroup_get(EventGroup *group, uint32_t *outBits)
 	*outBits = group->bits;
 	return 0;
 }
+
+int eventgroup_destroy(SObjStruct *heapCapability, EventGroup *group)
+{
+	group->lock.lock();
```

Comment:
What will happen to any threads waiting on this lock once we free `group`? I guess they will timeout and then crash? Oh well.